validator/sawtooth_validator/execution/processor_handlers.py

```diff
@@ -33,22 +35,25 @@ def __init__(self, processor_collection):
     def handle(self, connection_id, message_content):
         request = processor_pb2.TpRegisterRequest()
         request.ParseFromString(message_content)
+        max_occupancy = request.max_occupancy or DEFAULT_MAX_OCCUPANCY
```

Comment:
Suggest using an explicit check for `== 0` here.  It will be more clear about types